launcher/launcher_windows.go

```diff
@@ -3,17 +3,10 @@
 package launcher
 
 import (
-	"fmt"
 	"os/exec"
 )
 
 // OpenURI opens a given uri in a web browser.
-func OpenURI(uri string) {
-	cmd := exec.Command("start", uri)
-	err := cmd.Start()
-	if err != nil {
-		fmt.Errorf("%s\n", err)
-	} else {
-		err = cmd.Wait()
-	}
+func OpenURI(uri string) error {
+	return exec.Command("rundll32", "url.dll,FileProtocolHandler", uri).Start()
```

Comment:
Why not just use the "start" binary? This recognizes URI's and opens them in your default browser. Not sure why we need to call rundll32 here...